frontend/client-list/toolbar/clients-table-toolbar.component.tsx

```diff
@@ -122,4 +230,12 @@ type ClientsTableToolbarProps = {
   setPage(pageNum: number): void;
   setSearch(searchParse: SearchParseValues): any;
   fetchingClient: boolean;
+  selectedClients: any;
+  setSelectedClients: (client: object) => any;
+  modalIsOpen: boolean;
+  setModalIsOpen: any;
```

Comment:
Is using `any` here okay or should it be `(modalIsOpen: boolean) => any`

Reply:
should be `setModalIsOpen: (isOpen: boolean)`